### PR TITLE
Added is_signed flags to wheel/vehicle speed measurements.

### PR DESCRIPTION
**Pull request by adamshapiro0**

### CODE REVIEW

File: src/point_one/fusion_engine/messages/measurements.h [hunk 1]
```diff
@@ -318,7 +324,13 @@ struct alignas(4) VehicleSpeedMeasurement : public MessagePayload {
...
   GearType gear = GearType::UNKNOWN;
 
-  uint8_t reserved[3] = {0};
+  /**
+   * `true` if the wheel speeds are signed (positive forward, negative reverse),
+   * or `false` if the values are unsigned (positive in both directions).
+   */
+  bool is_signed = true;
+
+  uint8_t reserved[2] = {0};
```
Comment (ryancdougherty) [inline src/point_one/fusion_engine/messages/measurements.h:333]: Added 3 bytes?

Comment (ryancdougherty) [reply src/point_one/fusion_engine/messages/measurements.h:333]: Oh -- replaces 3 padding bytes

**Review by ryancdougherty — APPROVED** [review]
Looks good to me

File: src/point_one/fusion_engine/messages/measurements.h (hunk 1 above)
Comment (adamshapiro0) [reply src/point_one/fusion_engine/messages/measurements.h:333]: 2 bytes reserved: gear (1B), is signed (1B), reserved (2B)

**Pull request merged**